src/lambda_handler.py

```diff
@@ -1,24 +1,157 @@
-"""Simple AWS Lambda handler to verify functionality."""
+"""AWS Lambda handler to publish egress IPs from a provided list of AWS accounts."""
 
 # Standard Python Libraries
 from datetime import datetime, timezone
+from ipaddress import collapse_addresses, ip_network
 import logging
 import os
-from typing import Any, Optional, Union
+import re
+from typing import Any, Dict, Iterator, List, Optional, Set, Tuple, TypedDict, Union
 
 # Third-Party Libraries
-import cowsay
-import cowsay.characters
-
-# cisagov Libraries
-from example import example_div
+import boto3
 
 default_log_level = "INFO"
 logger = logging.getLogger()
 logger.setLevel(default_log_level)
 
 
-def failed_task(result: dict[str, Any], error_msg: str) -> None:
+def assume_role(role_arn: str, session_name: str) -> Tuple[str, str, str]:
```

Comment:
Would it be better to use a dictionary or [named tuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) here?

Reply:
Good suggestion, thanks!  Please see https://github.com/cisagov/publish-egress-ip-lambda/pull/4/commits/a9ec40caa6dbf52a8c2f10e9afd6942702c52de4.